lib/matplotlib/patches.py

```diff
@@ -1657,6 +1669,14 @@ def get_angle(self):
 
     angle = property(get_angle, set_angle)
 
+    def get_corners(self):
+        """
+        Return the corners of the ellipse bounding box, moving anti-clockwise
+        from the lower left corner defined before rotation.
```

Comment:
The first line really shouldn't be wrapping.
```suggestion
        Return the corners of the ellipse bounding box.
        
        The bounding box orientation is moving anti-clockwise
        from the lower left corner defined before rotation.
```